### PR TITLE
Feat/306 improve nav bar recipe card

### LINKED ISSUE #306: style(nav-bar): improve nav bar and common recipe card styling
### Suggestions from Dec 5 meeting

- [x] Nav bar styling
    - [x] Remove search bar and add an Search button to lead to search page
    - [x] Remove the category button
    - [x] Change order of buttons to, About Us -> Contribute Recipe -> Meal Planner -> Search
    - [x] Move About Us more to the right, away from Yummy Recipes
    - [x] Try different hover animations, see which one looks better like underline
    - [x] At some width, before yummy recipes overflows, change it to team logo and link it
- [x] Recipe card styling
    - [x] Add specific width for card
    - [x] Make shadow smaller for cards

### PR DESCRIPTION
**Pull request by keskinmbaha**
Finished all the suggested styling comments we had from the meeting.

Closes #306

### CODE REVIEW

File: source/components/common/nav-bar/common-nav-bar.css [hunk 1]
```diff
@@ -18,68 +18,35 @@
...
-.category-link {
-  display: none;
-  height: 50px;
-  line-height: 50px;
-  background-color: white;
-  box-shadow: 5px 5px 2px 1px rgba(0, 0, 255, 0.2);
-}
-.category-link:hover {
-  background-color: #ffd3d3;
+.div-links {
```
Comment (jbutera94) [inline source/components/common/nav-bar/common-nav-bar.css:36]: I'm wondering if `.nav-link-container` makes more sense as the name.

**Review by jbutera94 — CHANGES_REQUESTED** [review]
Looks good, left one small suggestion.

**Comment by keskinmbaha** [comment]
Have to fix the search page routing system.

File: source/components/common/nav-bar/common-nav-bar.html [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+  <div class="nav-link-container">
+    <a class="nav-link" href="#/recipes/contribute">Contribute Recipe</a>
+  </div>
+  <div class="nav-link-container">
+    <a class="nav-link" href="#/recipes/meal-planner">Meal Planner</a>
+  </div>
+  <div class="nav-link-container">
+    <a
+      class="nav-link"
+      href="#/recipes/search?query=&glutenFree=true&healthy=true&highProtein=true&vegan=true&vegetarian=true?page=1"
```
Comment (jbutera94) [inline source/components/common/nav-bar/common-nav-bar.html:19]: Changing the search URL to `#/recipes/search?query=&glutenFree=true&healthy=true&highProtein=true&vegan=true&vegetarian=true&page=1` should fix the routing issue.

**Review by jbutera94 — CHANGES_REQUESTED** [review]

**Review by jbutera94 — APPROVED** [review]
Looks good!

**Pull request merged**